### DISCUSSION TITLE
Cannot create/patch cluster-ca-cert secret

### DISCUSSION THREAD

**Discussion by ykizin** [strimzi/strimzi-kafka-operator#7582] [Q&A]
**Describe the bug**
After applying manifests at EKS by using ArgoCD we have some problems - cannot create a cluster with the error "
Operation: [patch] for kind: [Secret] with name: [prod-msa-kafka-cluster-ca-cert] in namespace: [kafka] failed."
ps: it's a new EKS cluster with a fresh operator setup

```
 Status:                                                                                                                                                                        │
│   Conditions:                                                                                                                                                                  │
│     Last Transition Time:  2022-11-05T15:36:36.580819Z                                                                                                                         │
│     Message:               Operation: [patch]  for kind: [Secret]  with name: [prod-msa-kafka-cluster-ca-cert]  in namespace: [kafka]  failed.                 │
│     Reason:                KubernetesClientException                                                                                                                           │
│     Status:                True                                                                                                                                                │
│     Type:                  NotReady                                                                                                                                            │
│   Observed Generation:     2                                                                                                                                                   │
│ Events:                    <none>
```
If I try to recreate the operator pod and the resource, I have the next error "An error has occurred." and next error "Operation: [patch] for kind: [Secret] with name: [prod-msa-kafka-cluster-ca-cert] in namespace: [kafka] failed."

I didn't find any problems with RBAC. Logs and manifest I attached to issue

[strimzi_report.zip](https://github.com/strimzi/strimzi-kafka-operator/files/9946619/strimzi_report.zip)

**To Reproduce**

Apply the ArgoCD application to set up the operator and CR

**Expected behavior**

Strimzi Operator create Kafka cluster

**Environment (please complete the following information):**
 - Strimzi version: 0.32
 - Installation method: Helm chart
 - Kubernetes cluster: Kubernetes 1.23
 - Infrastructure: Amazon EKS
 - Other: ArgoCD: 2.4.11, Kafka version: 3.2.3 (we try setup 3.2-3.3.1)

**Comment by scholzj** [comment]
The request to the Kubernetes API seems to be timing out. So you probably should check your Kubernetes cluster logs as to why. Could be some tooling you have which is intercepting it or some issue somewhere in your API server.
